addons/mail/static/src/core/common/chat_hub.js

```diff
@@ -16,12 +16,12 @@ export class ChatHub extends Component {
     static template = "mail.ChatHub";
 
     get chatHub() {
-        return this.store.chatHub;
+        return this.store && this.store.chatHub;
     }
 
     setup() {
         super.setup();
-        this.store = useService("mail.store");
+        this.store = useService("mail.store", (useStateService) => (this.store = useStateService));
```

Comment:
I would name variable `store2` or `s` even. This is necessary for technical reasons, it's best to reduce it's footprint on code as much as we can:
```js
this.store = useService("mail.store", (s) => (this.store = s));
```